platform/packages/finance/src/fractionable/coin.rs

```diff
@@ -37,27 +37,33 @@ mod test {
     };
 
     #[test]
-    fn safe_mul() {
+    fn checked_mul() {
         use crate::fractionable::Fractionable;
         assert_eq!(
             Coin::<SuperGroupTestC1>::new(30),
-            Coin::<SuperGroupTestC1>::new(3).safe_mul(&Percent::from_percent(1000))
+            Fractionable::<u32>::checked_mul(
```

Comment:
[everywhere] not sure we need the extended syntax when calling `checked_mul`. More concise and readable is if we call it on the first operand